.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=true
```

Comment:
Is that automatically created in your workspace? this file might not be needed because package-lock defaults to true already

Reply:
In my workspace, all apps are defaulted to have `package-lock=false` , and `save-exact=true`. 

Having, `package-lock=false` helps `^` is used, as it allows automatic update of apps, without manipulating the lock file. With lock file, it does not fetch the latest one from (our) secured Nexus repo. 

Anyways, I have reverted it, and published.

Reply:
ahh makes sense, thanks for sharing that!